examples/google-cloud-speech/output/google-cloud-speech/lib/google/cloud/speech/v1/credentials.rb

```diff
@@ -0,0 +1,41 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+require "googleauth"
```

Comment:
The generator should be able to produce client code that works with non-Google backends. So we'll need to add something to the input to reflect this. Maybe something like a `--google-auth` option?

This also contains Cloud specific things like the `README.md`

Reply:
The class this file defines inherits from a class in googleauth. Do you want to remove this file? Or represent this class a different way?

Reply:
I think this is fine as-is unless you folks have a desire to change it. 

My point wasn't about the output of the generator, but just to point out that we need to think about how we run/configure it. If the input protos are not google services then we don't want to add a dependency on a Google auth library.

Reply:
Is the license copyright added to every file also something that needs to be configurable in the generator?

Reply:
yes, and it should have a good default (like Apache) without any company names in it.